scripts/augment_train_noise.py

```diff
@@ -19,10 +19,28 @@ def main(args):
     # List files in that dir
     sound_paths = glob.glob(os.path.join(subdir, '**/*.wav'),
                             recursive=True)
-    print(f'Augmenting {subdir} files')
-    # Transform audio speed
-    augment_noise(sound_paths, 0.8)
-    augment_noise(sound_paths, 1.2)
+    # Avoid running this script if it already have been run
+    if len(sound_paths) == 60000:
+        print("It appears that augmented files have already been generated.\n"
+              " If the augmentation process went well the first time you ran "
+              "generate_librimix.sh please comment out the line relative to"
+              " augment_train_noise.py.\nSkipping data augmentation.")
```

Comment:
Let's not ask to comment the augmentation line. It's more confusing than helping IMO. Just keep the "It appears ... generated. Skipping data augmentation"

Reply:
Agree